### PR TITLE
sepolicy changes for fluentbit

### PR DESCRIPTION
**Pull request by patope**
There policy changes are need to make fluentbit to work on cluster

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -860,10 +860,10 @@ EOF
...
     allow container_t container_log_t:file { open read watch };
     allow container_t container_log_t:lnk_file read;
-    allow container_t var_log_t:dir { add_name write };
-    allow container_t var_log_t:file { create lock open read setattr write };
-    allow container_t var_log_t:dir remove_name;
-    allow container_t var_log_t:file unlink;
+    allow container_t var_log_t:dir { add_name write watch read remove_name };
+    allow container_t var_log_t:file { create lock open read setattr write getattr unlink };
+    allow container_t var_lib_t:dir { add_name write read remove_name };
+    allow container_t var_lib_t:file { create lock open read setattr write getattr unlink };
```
Comment (mysticaltech) [inline locals.tf:866]: @patope var_log_t should not be removed.

**Review by mysticaltech — CHANGES_REQUESTED** [review]

File: locals.tf (hunk 1 above)
Comment (patope) [reply locals.tf]: Now there are 3 separate rows for `var_log_t:dir` and `var_log_t:file` permissions

Comment (aleksasiriski) [reply locals.tf]: Why not combine all the perms of the same class into single line? I see that wasn't done in the master branch, but maybe it should? Correct me if I'm wrong.

Comment (mysticaltech) [reply locals.tf]: I want to do that too, will fix.

**Review by mysticaltech — APPROVED** [review]
Thanks for this @patope, and sorry for the delay, was a busy week. Merging now.

**Pull request merged**